test/Redux.spec.js

```diff
@@ -0,0 +1,47 @@
+import expect from 'expect';
+import Redux from '../src/Redux';
+
+const fakeState = { foo: 'bar' };
+
+function fakeStore() {
+  return fakeState;
+}
+
+describe('Redux', () => {
+
+  let redux;
+
+  beforeEach(() => {
+    redux = new Redux({ fakeStore });
+  });
+
+  it('should correctly initialize', () => {
+    expect(redux.state).toEqual({ fakeStore: fakeState });
+    expect(redux.listeners).toEqual([]);
+    expect(redux.dispatcher).toBeA('function');
+    expect(redux.dispatchFn).toBeA('function');
+  });
+
+  it('should subscribe to changes', done => {
+    redux.subscribe(() => {
+      const state = redux.getState();
+      expect(state).toEqual(fakeState);
```

Comment:
This test isn't sufficient because the updated state is the same as the initial state, since `fakeStore()` always returns the same thing.